src/scrapers/base.iced

```diff
@@ -105,8 +105,11 @@ class BaseScraper
     if rc is v_codes.OK
       try
         if opts.json
-          response.json().then (body) ->
+          response.json().then((body) ->
             cb err, rc, body
+          ).catch((err) ->
+            cb err, v_codes.BASE_HARD_ERROR
+          )
```

Comment:
unhandled promise rejection bug when we ask for json but the site has returned something unparseable

Reply:
I don't think BASE_HARD_ERROR is meant to be used as a real code, rather it's the value that marks the start of the hard error section. How about CONTENT_FAILURE?